src/librustc_feature/builtin_attrs.rs

```diff
@@ -47,7 +47,7 @@ pub enum AttributeType {
     /// Builtin attribute that may not be consumed by the compiler
     /// before the unused_attribute check. These attributes
     /// will be ignored by the unused_attribute lint
-    Whitelisted,
+    Allowlisted,
```

Comment:
This case is responsible for a significant part of renaming, but both the old and new name miss the point.

This should rather be `AssumeUsed` or something (except that the whole enum is pretty strange and should probably be refatored into something else - the crate level restriction and "used" assumptions are orthogonal).

Reply:
Yeah the usage of Whitelisted here has always bugged me.

A good side benefit of critically evaluating language used for inclusion is that you often end up with more precise language _anyway_ 😁

Reply:
I too was often confused by this existing name and couldn't figure out what it was supposed to mean.